src/middleware-wrapper.js

```diff
@@ -12,17 +12,18 @@ const middlewareWrapper = (config) => {
     fs.readFileSync(path.join(__dirname, '/public/index.html'))
       .toString()
       .replace(/{{title}}/g, config.title)
+      .replace(/{{port}}/g, config.port)
```

Comment:
so this is the port of the socket.io instance or the express server? I am confused 😅